front/src/applications/operationalStudies/components/MacroEditor/ngeToOsrd.ts

```diff
@@ -36,10 +40,13 @@ import type {
   LabelDto,
 } from '../NGE/types';
 
-const getNodeById = (nodes: NodeDto[], nodeId: number | string) =>
+const getNodeById = (nodes: NodeDto[], nodeId: number | string): NodeDto | undefined =>
```

Comment:
You don't need to specify this, ts knows what it is manipulating ;) (and it's -1 line of change !) 
```suggestion
const getNodeById = (nodes: NodeDto[], nodeId: number | string) =>
```

Reply:
Done in, "remove overtyping"